index.html

```diff
@@ -200,6 +200,8 @@
         openmct.install(openmct.plugins.Timelist());
         openmct.install(openmct.plugins.BarChart());
         openmct.install(openmct.plugins.ScatterPlot());
-        openmct.start();
+        document.addEventListener('DOMContentLoaded', function () {
+            openmct.start();
```

Comment:
do we need to do this to yamcs and viper?

Reply:
Most likely? Will need to test this against yamcs/viper with the new Playwright.